gito/core.py

```
@@ -76,12 +88,52 @@ def get_diff(
-            f"Using merge base: {mc.ui.cyan(merge_base.hexsha[:8])} ({merge_base.summary})"
-        )
+
+        # if branch is already an ancestor of "against", merge_base == branch ⇒ it’s been merged
+        if merge_base.hexsha == repo.commit(current_ref).hexsha:
+            # @todo: check case: reviewing working copy index in main branch #103
+            logging.info(f"Branch is already merged. ({ui.green(current_ref)} vs {ui.yellow(against)})")
+            merge_sha = repo.git.log(
+                '--merges',
+                '--ancestry-path',
+                f'{current_ref}..{against}',
+                '-n','1',
```

Flake8 (7.2.0)

[error] missing whitespace after ','

(E231)